baseapp-profiles/baseapp_profiles/permissions.py

```
@@ -27,16 +28,19 @@ def has_perm(self, user_obj, perm, obj=None):
                 return user_obj.has_perm(perm)
+           
 
         if perm == "baseapp_profiles.use_profile" and obj:
             if isinstance(obj, Profile):
                 return (
                     obj.owner_id == user_obj.id or obj.members.filter(user_id=user_obj.id).exists()
                 )
+           
 
         if perm == "baseapp_profiles.delete_profile" and obj:
             if isinstance(obj, Profile):
```

Ruff (0.8.0)

Use a single `if` statement instead of nested `if` statements

(SIM102)